tests/unit/dataactvalidator/test_b9_award_financial.py

```diff
@@ -55,6 +55,18 @@ def test_success_mismatched_year(database):
     assert number_of_errors(_FILE, database, models=[af, pa]) == 0
 
 
+def test_success_unknown_value(database):
+    """Program activity name/code as null"""
+    af_1 = AwardFinancialFactory(row_number=1, beginning_period_of_availa=2016, agency_identifier='test',
+                                 allocation_transfer_agency='test', main_account_code='test',
+                                 program_activity_name='test', program_activity_code='0000')
+    af_2 = AwardFinancialFactory(row_number=1, beginning_period_of_availa=2016, agency_identifier='test',
+                                 allocation_transfer_agency='test', main_account_code='test',
+                                 program_activity_name='test', program_activity_code='Unknown/Other')
```

Comment:
So the allowable values for the related JIRA ticket are as follows:

- `program_activity_name`: "Unknown/Other"
- `program_activity_code`: "0000"

Here, `program_activity_code` is being set to "Unknown/Other" so we're not testing what the SQL is testing (at least for success). `program_activity_code='Unknown/Other'` _should_ fail if i'm not mistaken